docs/dev/notebook-jobs-config-mapping.json

```diff
@@ -0,0 +1,19 @@
+{
+  "etl/gff": "docs/dev/jobs-config.json",
```

Comment:
This adds a lot of boilerplate. Can we default to `docs/dev/jobs-config.json` if a job config is not present in the dictionary? Then the dictionary can contain only the custom cases.